.gitignore

```diff
@@ -15,3 +16,23 @@ node_modules
 # Unit test / coverage reports
 .coverage
 htmlcov/
+
+# Temporary files
+*.swp
+*.swo
+*.tmp
+*~
+*.todo
+
+# OS generated files
+.DS_Store*
+thumbs.db
+Thumbs.db
+Desktop.ini
+
+# Editor files
+.vimcache
+.vscode
+*.sublime-workspace
+*.sublime-project
+.idea
```

Comment:
Please add a new line to the end of the file.

Reply:
@data-get line added, marking this as resolved.